src/response.php

```diff
@@ -141,9 +148,18 @@ public function send_response($content_type=null, $encode_options=null, $respons
 	
 	if ($this->http_status == self::STATUS_NO_CONTENT) {
 		return;
-	}
+	} 
 	
-	echo $response;
+	if ($content_type == self::CONTENT_TYPE_JSONP) {
+		if (empty($jsonp_callback)) {
+			$jsonp_callback = self::JSONP_CALLBACK_DEFAULT;
```

Comment:
So this generates a response of `JSON_CALLBACK({"foo":"bar"})`? I'm not so much into jsonp, but isn't there a common callback name? I guess capitalized and underscored is not really the default callback name :)